src/mscorlib/shared/System/IO/PathInternal.cs

```diff
@@ -106,5 +109,93 @@ internal static bool AreRootsEqual(string first, string second, StringComparison
                     length: firstRootLength,
                     comparisonType: comparisonType) == 0;
         }
+
+        /// <summary>
+        /// Try to remove relative segments from the given path (without combining with a root).
+        /// </summary>
+        /// <param name="skip">Skip the specified number of characters before evaluating.</param>
+        internal static string RemoveRelativeSegments(string path, int skip = 0)
+        {
+            Debug.Assert(skip > 0);
```

Comment:
Why have a default value for the 2nd parameter if it's invalid? It would never be valid to not specify an explicit value, therefore the default value is not useful.

Reply:
@danmosemsft its a mistake i will correct it to ```skip >= 0```